### PR TITLE
Minor: Updated core-pod.yml

### LINKED ISSUE #5: Create first version of core app pod
Create core app pod that will be based on busybox image so far

### PR DESCRIPTION
**Pull request by Oleksandr6676**
Created first version of core app pod
Closes #5

### CODE REVIEW

**Comment by JanMate** [comment]
Closes #6 

(Only for Github automation)

File: kubernetes/core-svc.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: busybox-service
```
Comment (JanMate) [inline kubernetes/core-svc.yml:4]: Could we use name 'busybox-svc' to have shorter name?

File: kubernetes/core-svc.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+kind: Service
+metadata:
+  name: busybox-service  
+  labels:
+    app: busybox
+spec:
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 8080
```
Comment (JanMate) [inline kubernetes/core-svc.yml:11]: Is this port number right? What does targetPort stands for?

File: kubernetes/core-svc.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: busybox-service  
+  labels:
+    app: busybox
+spec:
+  ports:
```
Comment (JanMate) [inline kubernetes/core-svc.yml:8]: I cannot see the type of the service. What is the type by default? Could we explicitly define it?

**Review by JanMate — CHANGES_REQUESTED** [review]
Congrats to very first k8s resources! 👍🏽 Let's improve them and the world will be better then! 👌🏽

File: kubernetes/core-svc.yml (hunk 2 above)
Comment (Oleksandr6676) [reply kubernetes/core-svc.yml:11]: Target port is the port on the pod to forward traffic, right? So I guess it should be 80 as well

File: kubernetes/core-svc.yml (hunk 3 above)
Comment (Oleksandr6676) [reply kubernetes/core-svc.yml:8]: ClusterIP is set by default. Yes, it is possible, do I need to change it?

Comment (JanMate) [reply kubernetes/core-svc.yml:9]: I believe ClusterIP suites for our purpose perfectly.

File: kubernetes/core-svc.yml (hunk 2 above)
Comment (JanMate) [reply kubernetes/core-svc.yml]: Exactly! TargetPort property refers to port of the container.

**Review by JanMate — APPROVED** [review]
Great first step! I'm looking forward when we have a CI to download all package and run it in a cluster.

**Pull request merged**